site/src/Librecores/ProjectRepoBundle/Util/GitHubUtil.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Librecores\ProjectRepoBundle\Util;
+
+/**
+ * Helper utility service, which provides methods for operating with GitHub projects.
+ */
+
+class GitHubUtil
+{
+
+    private $githubUrl;
+    private $shieldsUrl;
+    private $shieldsExt;
+    private $sourceRepoUrl;
+    private $issueTrackerUrl;
+
+    public function __construct($githubUrl, $shieldsUrl, $shieldsExt)
+    {
+        $this->githubUrl       = $githubUrl;
+        $this->shieldsUrl      = $shieldsUrl;
+        $this->shieldsExt      = $shieldsExt;
+        $this->sourceRepoUrl   = null;
+        $this->issueTrackerUrl = null;
```

Comment:
Yes these are nice to have, we should add them, at least the links to them (I suppose you mean /releases, /graph). For more info let's bite the bullet and use their api. I found a cool client library, more info on #80.